### DISCUSSION TITLE
quarkus-hal does not provide dependency to quarkus-resteasy-links

### DISCUSSION THREAD

**Discussion by humcqc** [quarkusio/quarkus#36226] [Q&A]
Hello,

I would like to use HAL in my application,
I follow this doc: https://quarkus.io/guides/resteasy.
But when I just use quarkus-hal extension I do not have access to @LinkResource annotation.
I need to manually add  quarkus-resteasy-links extension.

Should the doc be updated ? 
Or am I missing something ?

When i look at the reactive equivalent, annotation are different: LinkResource vs RestLink, is possible to keep one annotation ?

Thanks

**Comment by kinkkong** [comment]
Try here 
```
<dependency>
      <groupId>io.quarkus</groupId>
      <artifactId>quarkus-rest-links</artifactId>
    </dependency>
```
